pkgs/by-name/n-/n-m3u8dl-re/reverse-arr.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/src/N_m3u8DL-RE.Common/Entity/WebVttSub.cs b/src/N_m3u8DL-RE.Common/Entity/WebVttSub.cs
+index 6b6d284..b57fc24 100644
+--- a/src/N_m3u8DL-RE.Common/Entity/WebVttSub.cs
++++ b/src/N_m3u8DL-RE.Common/Entity/WebVttSub.cs
+@@ -201,7 +201,8 @@ public partial class WebVttSub
+             time += Convert.ToInt32(parts.Last().PadRight(3, '0'));
+             str = parts.First();
+         }
+-        var t = str.Split(':').Reverse().ToList();
++        var t = str.Split(':').ToList();
++        t.Reverse();
```

Comment:
I'm not familiar with c# but as per msft dotnet 9.0 docs, [array.Reverse](https://learn.microsoft.com/en-us/dotnet/api/system.array.reverse?view=net-9.0#system-array-reverse(system-array)) is void, so this is the correct way to call it. (same with 8.0)

Reply:
Is there an upstream PR for this patch? It might be nice to use a `fetchpatch` to pull that instead of vendoring it (although I'm not 100% confident what the policy for patches in Nixpkgs is).